charmstore.go

```diff
@@ -232,6 +233,94 @@ func (s *CharmStore) Latest(curls ...*charm.URL) ([]CharmRevision, error) {
 	return responses, nil
 }
 
+func (s *CharmStore) ListResources(ids ...*charm.URL) ([]ResourceResult, error) {
```

Comment:
This just follows how Latest works.  It's not how I'd do it, but it's not my codebase.